src/pages/workspace/workflows/WorkspaceWorkflowsPage.tsx

```diff
@@ -110,42 +137,64 @@ function WorkspaceWorkflowsPage({policy, betas, route}: WorkspaceWorkflowsPagePr
                 title: translate('workflowsPage.makeOrTrackPaymentsTitle'),
                 subtitle: translate('workflowsPage.makeOrTrackPaymentsDescription'),
                 onToggle: () => {
-                    // TODO will be done in https://github.com/Expensify/Expensify/issues/368335
+                    const isActive = policy?.reimbursementChoice === CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
+                    const newReimbursementChoice = isActive ? CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_MANUAL : CONST.POLICY.REIMBURSEMENT_CHOICES.REIMBURSEMENT_YES;
```

Comment:
@lakchote is this correct?

Reply:
> olicy?.reimbursementChoice don't persist after signout and sign

I'm seeing that too, but just on refresh and not consistently. Were you able to get some consistent repro steps? @ishpaul777

Reply:
We merged and deployed a fix for the API so I think this is fixed.